.gitignore

```diff
@@ -1,4 +1,6 @@
 node_modules
-public/assets/css/style.css*
 public/OpenFinD3FC.zip
-.idea
\ No newline at end of file
+.idea
+public/assets/fonts
+public/assets/css
+public/app.json
```

Comment:
It will only ignore the files in the assets folder, and we don't have anything of our own in there at the moment right?